.github/workflows/ci-windows-installed.yml

```diff
@@ -28,6 +30,13 @@ jobs:
         codec-aom: 'SYSTEM'
         codec-dav1d: 'SYSTEM'
         codec-rav1e: 'SYSTEM'
+        extra-cache-key: ${{ matrix.compiler }}
+    - name: Leave compiler as default
+      if: matrix.compiler == 'default'
+      run: echo "CMAKE_AVIF_FLAGS=\"\"" >> $GITHUB_ENV
+    - name: Set clang-cl as compiler
+      if: matrix.compiler == 'clang-cl'
+      run: echo "CMAKE_AVIF_FLAGS=-DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl" >> "$GITHUB_ENV"
```

Comment:
Nit: I suggest avoiding the `CMAKE_` prefix because cmake owns that namespace. For example, we can name this variable `AVIF_CMAKE_FLAGS`.

Reply:
Done